src/util-dpdk-bonding.c

```diff
@@ -52,7 +52,10 @@ uint16_t BondingMemberDevicesGet(
 {
 #ifdef HAVE_DPDK_BOND
 #if RTE_VERSION >= RTE_VERSION_NUM(23, 11, 0, 0)
+#pragma GCC diagnostic push
```

Comment:
git message should reference https://redmine.openinfosecfoundation.org/issues/7009